.github/workflows/conformance.yml

```
@@ -41,8 +41,8 @@ jobs:
       - name: Clone gateway-api
         uses: actions/checkout@v2
         with:
-          repository: "nathancoleman/gateway-api"
-          ref: "eventually-consistent-conformance"
+          repository: "hashicorp/gateway-api"
+          ref: "conformance/reference-policy-60s-timeout"
```

```suggestion
          ref: "60s-time-to-consistency"
```

I pushed a branch to our hashicorp fork that's name better indicates what it is